### ISSUE TITLE
Windows 10 support - library link failure

### ISSUE THREAD

**Issue by talavinari** [nschloe/pygalmesh#58]
Hello, 
TL;DR  - does pygalmesh support on windows (10) machines with python 3.6.6?

Here is my experience trying to run it on windows  **without success** :( 
after "pip3 install -U pygalmesh without success" failed on my python venv,
1. I cloned the source code
2. I installed CGAL (step by step instructions using Vcpkg Library Manager)
3. I downloaded Eigen and added "EIGEN_INCLUDE_DIR" env variable with the library path.
4. I ran python setup.py install and it failed on "missing CGAL libraries" , so I modified the setup.py file to be with my installation directory of CGAL:  (**added code in bold**)
  include_dirs=[
            os.environ.get("EIGEN_INCLUDE_DIR", "/usr/include/eigen3/"),
			**"C:\\dev\\vcpkg\\installed\\x86-windows\\include\\"**,
            # Path to pybind11 headers
            get_pybind_include(),
            get_pybind_include(user=True),
        ],

5. And now I can't overcome the following issue:

**LINK : fatal error LNK1181: cannot open input file 'stdc++.lib'
error: command 'D:\\Microsoft Visual Studio 2017\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX86\\x64\\link.exe' failed with exit status 1181**

I guess that this is the "libraries" installation part in the file :  
 **libraries=["stdc++", "gmp", "mpfr"]**

It seems that I don't really have these libraries, and I'm not sure how to install them.
Not sure if its relevant - but I've found here https://docs.python.org/3/distutils/setupscript.html that I can add library_dirs=[""]  to specify where to find these libs (but I don't really have them ...).

I tried to download them using MinGW but not sure how to compile or use them.

The same issue persists with VS 2019 and with gmp or mpfr as the "missing library" if I change their order in the setup.py file.

Any idea how to solve it ?
Many thanks in advance for your help !

**Comment by nschloe** [comment]
Well, you definitely need gmp and mpfr to compile. I have no idea if they have support for windows. Perhaps the CGAL mailing list is a better place to ask.

**Comment by talavinari** [comment]
OK will try there, thanks :)

**Comment by acfaruk** [comment]
If you install CGAL with the windows installer the libraries gmp and mpfr are installed as well. You would find them in the folder `INSTALL_PATH\auxiliary\gmp`.

I was able to make it work on windows, but I got an assertion error in one of the tests:
```
__________________________________________________________________________________ test_volume_from_surface ___________________________________________________________________________________ 

    def test_volume_from_surface():
        this_dir = os.path.dirname(os.path.abspath(__file__))
        mesh = pygalmesh.generate_volume_mesh_from_surface_mesh(
            os.path.join(this_dir, "meshes", "elephant.vtu"),
            facet_angle=25.0,
            facet_size=0.15,
            facet_distance=0.008,
            cell_radius_edge_ratio=3.0,
            verbose=False,
        )
    
        tol = 1.0e-3
        assert abs(max(mesh.points[:, 0]) - 0.357612477657) < tol
        assert abs(min(mesh.points[:, 0]) + 0.358747130015) < tol
>       assert abs(max(mesh.points[:, 1]) - 0.496137874959) < tol
E       assert 0.014181661246307764 < 0.001
E        +  where 0.014181661246307764 = abs((0.4819562 - 0.496137874959))
E        +    where 0.4819562 = max(array([-0.19258097, -0.09461719, -0.3258206 ,  0.06994132, -0.1434645 ,\n       -0.11351405,  0.05553488,  0.0659714 , ...5258,  0.37136012, -0.1880949 , -0.45568204,\n       -0.10133088,  0.07262085, -0.20886302, -0.19134675], dtype=float32))

test\test_volume_from_surface.py:21: AssertionError
```

**Comment by talavinari** [comment]
Thanks @acfaruk , all other functionality works as expected on you win machine ?

**Comment by acfaruk** [comment]
I later switched to another library, but the few examples I ran, were running as expected.

**Comment by nschloe** [comment]
I suppose this is dealt with now. Feel free to reopen if not.
